Source/dolphin-memory-engine.vcxproj

```diff
@@ -57,7 +57,7 @@
       <AdditionalIncludeDirectories>$(QTDIR)include;$(QTDIR)include\QtCore;$(QTDIR)include\QtGui;$(QTDIR)include\QtWidgets</AdditionalIncludeDirectories>
       <ObjectFileName>$(IntDir)%(RelativeDir)</ObjectFileName>
       <PreprocessorDefinitions>WIN32;WIN64;QT_DLL;QT_CORE_LIB;QT_GUI_LIB;QT_WIDGETS_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <LanguageStandard Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">stdcpp17</LanguageStandard>
+      <LanguageStandard Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">stdcpp20</LanguageStandard>
       <LanguageStandard_C Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">stdc17</LanguageStandard_C>
```

Comment:
Do we care to have this declared / should we update it for the heck of it?

Reply:
Actually, it looks like `LanguageStandard_C` is ignored if LanguageStandard is defined anyway. So we can remove that line entirely.

Reply:
Note that `stdc20` is not a thing at this time, I think. Whether the field can be removed, I don't know.

Reply:
Considering its not related to what C++ ver we are using, no reason to address it in this PR.